### ISSUE TITLE
Change tab color (MacOS)

### ISSUE THREAD

**Issue by martinnov92** [klauscfhq/hyper-pokemon#46]
Hello,

first I wanna thank you for a beautiful theme, second I would like to know if it is possible to change tab color, because I really hate that grey color there.

Thank you

<img width="1089" alt="Snímek obrazovky 2019-05-06 v 20 39 32" src="https://user-images.githubusercontent.com/15011249/57248776-88197400-7043-11e9-824b-6b813cc50d03.png">

**Comment by soraWontForget** [comment]
What does your .hyper.js config file look like? Do you have unibody option enabled? Did you have another theme installed previously or have you made any changes to the "css" section in your .hyper.js config file? I had the issue with tabs not being being the same color as the current theme's unibody color also. I fixed it by removing the "css" configurations from another theme in my .hyper.js config file.

**Comment by martinnov92** [comment]
I have nothing in `css` section and even after I enabled `unibody` in config file tabs look the same :/

**Comment by SAGARSURI** [comment]
Even I am facing the same issue.

**Comment by LucaSoato** [comment]
Yup, solved by deleting ~/.hyper.js and installing again, probably issue was caused by another theme.
Thank you guys!

**Comment by ghost** [comment]
@martinnov92 @SAGARSURI I fixed this problem. Please try it!
https://github.com/klaussinani/hyper-pokemon/pull/51#issuecomment-560233962
